monai/bundle/scripts.py

```diff
@@ -489,6 +490,12 @@ def patch_bundle_tracking(parser: ConfigParser, settings: dict):
                     handlers.append(v)
         elif k not in parser:
             parser[k] = v
+    # save the executed config into file
+    timestamp = time.strftime('%Y%m%d_%H%M%S')
+    filename = parser["output_dir"] + f"/config_{timestamp}.json"
```

Comment:
Hi @Nic-Ma ,
    The filename var would be ` $@bundle_root + '/eval'/config_20221207_064559.json`, if run with spleen_ct bundle. And in line 498, it would try to save the config with this name.

Thanks,
Bin